File: example/lib/cmed_user.dart
class CMEDUser {
  int? id;
  String? gender;
  int? ageInDays;
  int? birthDate;
  double? heightInCm;
  double? weightInKg;

  CMEDUser(this.id, this.gender, this.ageInDays, this.birthDate,
      this.heightInCm, this.weightInKg);

  CMEDUser.fromJson(Map<String, dynamic> json) {
    id = json['id'];
    gender = json['gender'];
    ageInDays = json['ageInDays'];
    birthDate = json['birthDate'];
    heightInCm = json['heightInCm'];
    weightInKg = json['weightInKg'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = <String, dynamic>{};
    data['id'] = id;
    data['gender'] = gender;
    data['ageInDays'] = ageInDays;
    data['birthDate'] = birthDate;
    data['heightInCm'] = heightInCm;
    data['weightInKg'] = weightInKg;
    return data;
  }
}

File: android/src/main/kotlin/com/cmedhealth/flutter/bmi/cmed_bmi_devices_lib/CMEDUser.kt
package com.cmed.plugin.lib.flutter.cmed_health_flutter_devices_lib.utils
import androidx.annotation.Keep

@Keep
class CMEDUser(
    private var id: Long?,
    private var gender: String?,
    var ageInDays: Int,
    var birthDate: Long,
    var heightInCm: Double,
    var weightInKg: Double
) {

    fun getGenderIndex(): Int? {
        return GenderEnum.getGenderEnum(gender.toString())?.index
    }

    fun setGender(gender: String?) {
        this.gender = gender
    }

    override fun toString(): String {
        return "User{id=$id, sex=$gender, age=$ageInDays,  birthDate=$birthDate, height=$heightInCm, weight=$weightInKg}"
    }
}